src/serial.rs

```diff
@@ -869,8 +869,17 @@ macro_rules! usart_var_clock {
 }
 
 cfg_if::cfg_if! {
-    if #[cfg(any(feature = "svd-f301", feature = "svd-f3x4"))] {
-        usart_var_clock!([(1,2)]);
+    if #[cfg(all(
+        any(
+            feature = "mem-4",
+            feature = "mem-6",
+            feature = "mem-8",
```

Comment:
Is this feature constraint not describable via the `svd-*` features? I mean the `mem-*` features do work, but are rather intended to distinguish memory differences and this seems not related.

Reply:
I used them because, for example f302x6/8 are affected, while f302xb/c/d/e are not.

Reply:
Hm. Yeah, this is not representable with the `svd-*`-features. But I would suggest to used `stm32f303xb`, `stm32f303xc` ... so that it is more obvious, when reading it :)

Reply:
> (Fig 12+13 vs. Fig 14 of RM0365)

Thanks for that info! I can see the difference. So I think the implementation is fine. Will still look at it later today :)